docs/application/vstools-mac/README.md

```diff
@@ -0,0 +1,147 @@
+# Installing Visual Studio for Mac Extension for Tizen
+
+## Prerequisites
+
+To work with Visual Studio for Mac Extension for Tizen, your computer must have:
+
+- At least 5.6 GB of available disk space
+- macOS Mojave 10.14: Community, Professional, and Enterprise.	
+- Visual Studio Mac 2019 (Community, Professional, Enterprise) Version 8.2 and above.
+
+- Java Development Kit (JDK)
+
+  You must install Oracle Java Development Kit (JDK) 8 or OpenJDK 10 to use 
+Tizen Baseline SDK. Make sure you download and install the EXACT version.
+
+  - [Oracle Java Development Kit(JDK) 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and [Java for OS X 2015-001](https://support.apple.com/kb/DL1572)
+ 
+  - OpenJDK 10 and OpenJFX [OpenJDK 10 and OpenJFX Installation Guide](https://developer.tizen.org/development/articles/openjdk-10-and-openjfx-installation-guide#install-openjdk-for-macos).
+
+For Oracle JDK, Java for OS X 2015-001 is required to support legacy Java software on macOS.
+
+<a name="setup"></a>
+## Visual Studio for Mac Extension for Tizen Installation
+
+You must install the **Visual Studio for Mac Extension for Tizen** and the **Baseline SDK** to use the Tizen SDK tools.
+
+### Installing the Extension
+
+The **Visual Studio for Mac Extension for Tizen** can be installed by following the below steps.
+
+1. In the Visual Studio Mac IDE menu, go to **Visual Studio** tab and click on **Extensions...** option in the menu.
+
+   ![Install extension](media/install-extension1.png)
+
+2. Extension Manager dialog is opened. In that click on **Gallery** tab which is present at the top left.
+
+	![Extension Manager](media/install-extension2.png)
+   
+3. Click on the **Refresh** button to get the latest extensions.
+
+	![Browse Extension](media/install-extension3.png)
+	
+4. After the extension list is refreshed, Expand the **IDE Extensions** list present in the **Gallery** tab and select the 
+   **Visual Studio for Mac Extension for Tizen** option and click on **Install...** button present at bottom right.
+
+	![Install Popup](media/install-extension4.png)
+	![Install Popup1](media/install-extension5.png)
+
+5. After **Install...** button is clicked. A pop-up window comes up, Click on **Install...** button. 	
+      
+        
+	![Install Popup2](media/install-extension6.png) 
+
+   The extension is installed. Please restart the IDE once the installation is completed.   
+   
+   After successful installation, **Tizen** option can be seen in the project wizard.
+   
+   ![Project Wizard](media/install-extension7.png)
+
+### Installing the Tizen Baseline SDK
+
+After installing the Visual Studio Mac Extension for Tizen, you must set up the Tizen baseline SDK:
+
+1. In the Visual Studio Mac IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
+2. Select **Install new Tizen SDK**.
+
+> **Note**
```

Comment:
Please insert two spaces at the end of this phrase or insert an empty line following this line.

Reply:
Done